.travis.yml

```diff
@@ -68,7 +68,7 @@ script:
   - docker exec -t alica-test bash -c "
     source /opt/ros/$ROS_DISTRO/setup.bash &&
     cd ~/catkin_ws &&
-    ./src/alica-supplementary/alica_tracing/scripts/install.sh &&
+    ./src/alica/alica-supplementary/alica_tracing/scripts/install.sh &&
```

Comment:
Do we really need this? Alica and supplementary tests can supply dummy tracing. we don't need jaeger tracing in tests.

Reply:
this is fixed in https://github.com/rapyuta-robotics/alica/pull/184